React/src/pages/employees/helper.ts

```diff
@@ -0,0 +1,84 @@
+import { FilterInputType } from "../../components/table/filter/TableFilter";
+import { HeadCell } from "../../components/table/sort/SortedTableHeader";
+import { Employee } from "../../types/model/Employee";
+
+export const headCells: Array<HeadCell<Employee>> = [
+    {
+        type: 'string',
+        id: 'firstName',
+        label: 'Imię',
+        disablePadding: false,
+        numeric: false,
+    },
+    {
+        type: 'string',
+        id: 'lastName',
+        label: 'Nazwisko',
+        disablePadding: false,
+        numeric: false,
+    },
+    {
+        type: 'string',
+        id: 'email',
+        label: 'Email',
+        disablePadding: false,
+        numeric: false,
+    },
+    {
+        type: 'string',
+        id: 'status',
+        label: 'Status',
+        disablePadding: false,
+        numeric: false,
+    },
+    {
+        type: 'string',
+        id: 'roles',
+        label: 'Role',
+        disablePadding: false,
+        numeric: false,
+    },
```

Comment:
We shouldn't display enum. 
Here you can se how Patryk translated enum https://github.com/s20146/PRO/blob/489e8837ec9476b211d004690ee1aefbe7d90ebc/React/src/pages/orders/helper.ts#L53